homeassistant/components/mqtt/vacuum/schema_legacy.py

```diff
@@ -92,6 +105,8 @@
 DEFAULT_RETAIN = False
 DEFAULT_SERVICE_STRINGS = services_to_strings(DEFAULT_SERVICES, SERVICE_TO_STRING)
 
+TEMPLATE_ERROR = "{#TEMPLATE_ERROR#}"
```

Comment:
I tend to use `object()` for unique sentinel objects. That also has the nice property of never being able to be in any other value domain.

Reply:
Used a more uniq alternative. But it must be  `str`.

Reply:
That's the benefit of a StrEnum. You get a string, but you can still check with `is` rather than `==`